app/konnect/org-management/sso.md

```diff
@@ -0,0 +1,176 @@
+    * The path does not require a slash (`/`).
+
+1. After clicking Save, close the configuration dialog and click Enable on your OIDC provider.
+
+## Test and apply the configuration
+
+{:.important}
+> **Important:** Keep built-in authentication enabled while you are testing IdP authentication. Only disable built-in authentication after successfully testing IdP authentication.
+
+You can test the SSO configuration by navigating to the login URI based on the organization login path you set earlier. For example: `https://cloud.konghq.com/login/examplepath`, where `examplepath` is the unique login path string set in the steps above. 
+
+If your configuration is set up correctly, you will see the IdP sign-in page.
+
+You can now manage your organization's user permissions entirely from the IdP
+application.
+
+## Advanced Settings
+
+You can configure custom IdP-specific behaviors in the **Advanced Settings** of the OIDC configuration form. The following options are available:
+
+1. **Scopes**: Specify the list of scopes {{site.konnect_short_name}} requests from the IdP. By default, {{site.konnect_short_name}} requests the `openid`, `email`, and `profile` scopes. The `openid` scope is required and cannot be removed.
+2. **Claim Mappings**: Customize the mapping of required attributes to a different claim in the `id_token` {{site.konnect_short_name}} receives from the IdP. By default, {{site.konnect_short_name}} requires three attributes: Name, Email, and Groups. The values in these attributes are mapped as follows:
+    - `name`: Used as the {{site.konnect_short_name}} account's `full_name`.
+    - `email`: Used as the {{site.konnect_short_name}} account's `email`.
+    - `groups`: Used to map users to teams defined in the team mappings upon login.
+{% endnavtab %}
+{% navtab SAML %}
+
+The {{site.konnect_short_name}} SAML integration allows you to configure various identity providers. While technically any SAML-compliant provider can be used, the following have been verified:
+
+* Okta 
+* Azure Active Directory
+* Oracle Identity Cloud Service 
+* Keycloak
+
+1. Log in to {{site.konnect_saas}}, click {% konnect_icon organizations %} **Organization**, and then select **Auth Settings**.
+
+1. Click **Configure provider** under **SAML**. 
+
+1. Enter the **Metadata URL** from your IdP in the **IDP Metadata URL** field.
+
+1. In the **Login Path** field, enter the unique string that matches the one in Okta. For example: `examplepath`.
+
+   {{site.konnect_short_name}} uses this string to generate a custom login
+   URL for your organization.
+
+   Requirements:
+    * The path must be unique across all {{site.konnect_short_name}} organizations.
+      If your desired path is already taken, you must choose another one.
+    * The path can be any alphanumeric string.
+    * The path does not require a slash (`/`).
+
+1. After clicking **Save**, configure the SP Entity ID and Login URL on your SAML IdP.
+
+## Test and apply the configuration
```

Comment:
You can't nest headers inside tabs, they break. Notice the duplicate headings here, and the incorrect nesting: 

<img width="241" alt="Screenshot 2024-07-31 at 3 30 02 PM" src="https://github.com/user-attachments/assets/ef52e97a-ac80-432e-9fdc-833379c2da1b">

Anchors also break. For example, https://deploy-preview-7481--kongdocs.netlify.app/konnect/org-management/sso/#advanced-settings won't work, and https://deploy-preview-7481--kongdocs.netlify.app/konnect/org-management/sso/#test-and-apply-the-configuration can only take you to one of the duplicate sections.

You can take one of two approaches here:
* Don't use headings for these sections. "Test" becomes another set of steps, but without a heading.
* Each heading gets a tab group.

I'm partial to each heading having its own tab group. So it would look something like this:

> ## Set up SSO in Konnect
> tab OIDC | tab SAML
> steps
> in the OIDC tab, there's an advanced settings table that doesn't get its own heading 
> 
> ## Test and apply the configuration
> tab OIDC | tab SAML

The nice thing is, once they have selected a tab, all tabs with the same name on the page will open as well. So if they're looking at SAML, the next section will also be open to SAML.